src/org/zaproxy/zap/extension/requester/ZapAddOn.xml

```diff
@@ -0,0 +1,17 @@
+<zapaddon>
+	<name>Requester</name>
+	<version>1</version>
+	<description>Request numbered panel.</description>
+	<author>Surikato</author>
+	<url></url>
+	<changes/>
+	<extensions>
+	    <extension>org.zaproxy.zap.extension.requester.ExtensionRequester</extension>
+	</extensions>
+	<ascanrules/>
+	<pscanrules/>		
+	<filters/>
+	<files/>
+	<not-before-version>2.4.0</not-before-version>
```

Comment:
This probably should be 2.4.2 as thats the version of the ZAP jar that it will get built with.